### ISSUE TITLE
[OTHER] Styling the chat control

### ISSUE THREAD

**Issue by AlexandreSajus** [Avaiga/taipy#1740]
### What would you like to share or ask?

Request from one of our customers:

How does one customize the chat control to respect the following requirements:
- Messages from all users need to be aligned on the left.
- The username should be aligned vertically with the user image, within the message box, and in bold.
- The message should be right below and aligned horizontally with the username.

Basically, like this image:
![image](https://github.com/user-attachments/assets/5035b1bf-00d0-4ef2-bda9-29938ad96feb)

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

**Comment by FredLL-Avaiga** [comment]
to get the user chat left align
```
.taipy-chat-sent {
    justify-content: flex-start;
}

```

**Comment by FredLL-Avaiga** [comment]
Here are the other changes to have something that looks like it
```
.taipy-chat-sent {
    justify-content: flex-start;
}
.taipy-chat-sent .MuiStack-root .MuiStack-root .MuiBox-root:last-child {
    border-radius: var(--border-radius) var(--border-radius) 0 0;
    font-size: inherit;
    background-color: var(--color-paper);
    width: 100%;
    margin-right: 0.7em
}

.taipy-chat-sent .MuiStack-root .MuiStack-root:last-child .MuiPaper-root {
    border-radius: 0 0 var(--border-radius) var(--border-radius);
}

.taipy-chat-sent .MuiStack-root .MuiStack-root:last-child .MuiPaper-root:before {
    content: none;
}


.taipy-chat-received .MuiStack-root .MuiStack-root .MuiBox-root:last-child {
    border-radius: var(--border-radius) var(--border-radius) 0 0;
    font-size: inherit;
    background-color: var(--color-paper);
    width: 100%;
}
.taipy-chat-received .MuiStack-root .MuiStack-root:last-child .MuiPaper-root {
    border-radius: 0 0 var(--border-radius) var(--border-radius);
}

```

**Comment by FredLL-Avaiga** [comment]
you can align the name by adding `padding-left: 0.7em `

But that will be in the component soon

**Comment by FredLL-Avaiga** [comment]
With the new layout of the chat component
```
/* left align sender block*/
.taipy-chat-sent {
    justify-content: flex-start;
}

/*sender avatar*/
.taipy-chat-sent .MuiStack-root .MuiBox-root {
    padding-top: unset;
}

/*sender user name*/
.taipy-chat-sent .MuiStack-root .MuiStack-root .MuiBox-root {
    border-radius: var(--border-radius) var(--border-radius) 0 0;
    font-size: inherit;
    background-color: var(--color-paper);
    width: 100%;
    margin-right: 0.7em;
}

/* sender message */
.taipy-chat-sent .MuiStack-root .MuiStack-root:last-child .MuiPaper-root {
    border-radius: 0 0 var(--border-radius) var(--border-radius);
    margin-right: unset;
}

/* remove sender message arrow*/
.taipy-chat-sent .MuiStack-root .MuiStack-root:last-child .MuiPaper-root:before {
    content: none;
}

/*sender avatar*/
.taipy-chat-received .MuiStack-root .MuiBox-root {
    padding-top: unset;
}

/* received user name*/
.taipy-chat-received .MuiStack-root .MuiStack-root .MuiBox-root {
    border-radius: var(--border-radius) var(--border-radius) 0 0;
    font-size: inherit;
    background-color: var(--color-paper);
    width: 100%;
}

/* received message*/
.taipy-chat-received .MuiStack-root .MuiStack-root .MuiPaper-root {
    border-radius: 0 0 var(--border-radius) var(--border-radius);
}

```

**Comment by FredLL-Avaiga** [comment]
About styling chat @FabienLelaquais
